src/components/Person/Person.jsx

```diff
@@ -1 +1,25 @@
-// Write code here
+import React from 'react';
+
+const Person = ({ person }) => (
+  <>
+    <h2 className="Person__name">
+      {`My name is ${person.name}`}
+    </h2>
+
+    {person.age && (
+      <p className="Person__age">{`I am ${person.age}`}</p>
+    )}
+
+    {person.isMarried ? (
+      <p className="Person__partner">
+        {`
+        My ${person.sex === 'm'
+          ? `wife's`
+          : `husband's`} name is ${person.partnerName}
+        `}
+      </p>
+    ) : 'I am not married'}
```

Comment:
too complicated, better create a variable in component with this condition and render it